deb/build/debian/jenkins.init

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env /lib/init/init-d-script
```

Comment:
See [`init-d-script(5)`](https://manpages.debian.org/buster/sysvinit-utils/init-d-script.5.en.html) for more information about this library.